src/pages/People.tsx

```diff
@@ -0,0 +1,64 @@
+// import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { PeopleItem } from '../components/PeopleItem';
+import { Person } from '../types';
+// import { getPeople } from '../api';
+
+export const People: React.FC<{
+  peopleFromServer: Person[] | undefined;
+  errorMessage: string;
+}> = ({ peopleFromServer, errorMessage }) => {
+  const { slug = '' } = useParams();
+
+  return (
+    <div className="container">
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {errorMessage.length > 2 && (
+            <p
+              data-cy={
+                errorMessage.length === 33
```

Comment:
use variables to describe 'magical' values

Reply:
I use errorMessage as exciter. When this variable have data, error starts.

Reply:
So you can write `!!errorMessage.length` - it means variable have data
Right now your entry looks unclear, and our priority is to learn how to write code that works and that other developers (for the team) understand